src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -2185,7 +2185,7 @@ void emitter::emitInsSve_R_R(instruction     ins,
         case INS_sve_uunpkhi:
         case INS_sve_uunpklo:
             assert(insScalableOptsNone(sopt));
-            assert(insOptsScalableAtLeastHalf(opt));
```

Comment:
Does doing this break the unit tests? I would hope there are some H,S,D tests that need switching to B,H,S